<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QQProxyAuthDialog</class>
 <widget class="QDialog" name="QQProxyAuthDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proxy Authentication</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="iconLabel">
     <property name="text">
      <string>ICON</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="introLabel">
     <property name="text">
      <string>Connect to proxy</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="usernameLabel">
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="userNameLineEdit"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="passwordLabel">
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
